docs/ruleset_functions/are_all_terminal_types_four_pipe(hvac.id).md

```diff
@@ -0,0 +1,24 @@
+# are_all_terminal_types_four_pipe
```

Comment:
Function related to how terminal type is defined.

Reply:
Based on the latest decisions this function is obsolete.